.env.example

```diff
@@ -15,3 +15,13 @@ CONTENTFUL_DELIVERY_API_TOKEN=
 
 # Your current space Content Preview API access token: https://www.contentful.com/developers/docs/references/content-preview-api/
 CONTENTFUL_PREVIEW_API_TOKEN=
+
+# Documentation: https://contentful.atlassian.net/wiki/spaces/PROD/pages/4138500311/Templates+-+Segment+tracking
+# Write Key for Starter Templates: https://app.segment.com/contentful-staging/sources/segment_starter_templates/settings/keys
+NEXT_PUBLIC_SEGMENT_WRITEKEY=
+
+# Source ID for Starter Templates: https://app.segment.com/contentful-staging/sources/segment_starter_templates/settings/keys
```

Comment:
We'd have to change this comment - it's the ID of the tracking plan protocol, not the source ID